server/models/household.js

```diff
@@ -0,0 +1,20 @@
+let mongoose = require('mongoose');
+let student = require('student.js')
+let Schema = mongoose.Schema;
+
+let HouseHoldSchema = new Schema({
+    students:
+        [student],
```

Comment:
nitpick: try formatting this similarly to the others with the opening bracket on the same line, and ending bracket on the last.

students: [
    student
],
id: {
    ...
}